code/modules/admin/verbs/adminhelp.dm

```diff
@@ -91,8 +91,8 @@ var/list/adminhelp_ignored_words = list("unknown","the","a","an","of","monkey","
 			//Options bar:  mob, details ( admin = 2, dev = 3, mentor = 4, character name (0 = just ckey, 1 = ckey and character name), link? (0 no don't make it a link, 1 do so),
 			//		highlight special roles (0 = everyone has same looking name, 1 = antags / special roles get a golden name)
 
-	var/mentor_msg = "\blue <b><font color=red>Request for Help: </font>[get_options_bar(mob, 4, 1, 1, 0)][ai_cl]:</b> [msg]"
-	msg = "\blue <b><font color=red>Request for Help:: </font>[get_options_bar(mob, 2, 1, 1)][ai_cl]:</b> [msg]"
+	var/mentor_msg = "<font color='blue'><b><font color=red>Request for Help: </font>[get_options_bar(mob, 4, 1, 1, 0)][ai_cl]:</b> [msg]</font>"
+	msg = "<font color='blue'><b><font color=red>Request for Help:: </font>[get_options_bar(mob, 2, 1, 1)][ai_cl]:</b> [msg]</font>"

```

Comment:
This is formatted weirdly because the person who originally wrote this was a dingus. Please reformat it less weirdly since we might as well do that if we're editing these lines.

Here's a corrected version.

```
var/mentor_msg = "<b><font color=red>Request for Help: </font><font color='blue'>[get_options_bar(mob, 4, 1, 1, 0)][ai_cl]:</b> [msg]</font>"
msg = "<b><font color=red>Request for Help:: </font><font color='blue'>[get_options_bar(mob, 2, 1, 1)][ai_cl]:</b> [msg]</font>"
```